.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: node_js
+node_js:
+  - "8"
+  - "10"
+
+before_install:
+  - travis_retry npm install
+
+script:
+  - npm test
```

Comment:
![EOF](https://i.redd.it/tn1obmdpoiw11.png)

Reply:
I fixed it 

![](https://media1.tenor.com/images/e9d5a73338a616c080d116c3b39f6894/tenor.gif?itemid=5681156)